index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="http://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.min.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav class="navbar">
+            <ul>
+                <li class="nav-logo"><a href="#">My Logo</a></li>
+                <li><img src="images/Menu.svg" alt="menu image" class="menu-image"></li>
+            </ul>
+        </nav>
+    </header>
+    <div class="flex-container">
```

Comment:
- [ ] OPTIONAL: You may consider this tag more appropriately to be a main-tag. :pray: